src/ansys/dpf/core/operators/mapping/fft.py

```diff
@@ -269,26 +289,47 @@ def inplace(self):
         return self._inplace
 
     @property
-    def calc_conjugated_coef(self):
-        """Allows to connect calc_conjugated_coef input to the operator.
+    def force_fft_points(self):
+        """Allows to connect force_fft_points input to the operator.
```

Comment:
```suggestion
        """Connects force fft points input to the operator.
```
Question: Could this description be written to "Forcibly connects fft points to the operator."